Chap_API_Job_Mgmt.tex

```diff
@@ -916,14 +907,11 @@ \subsection{\code{PMIx_Log}}
 \argin{ndirs}{Number of elements in the \refarg{directives} array (\code{size_t})}
 \end{arglist}
 
-Return codes are one of the following:
-
+\returnstart
 \begin{constantdesc}
-    \item \refconst{PMIX_SUCCESS} The logging request was successful.
     \item \refconst{PMIX_ERR_BAD_PARAM} The logging request contains at least one incorrect entry.
```

Comment:
Revision Exception:

```suggestion
```

Remove the bulleted list and replace with `\returnsimple`.